platform-sdk/swirlds-common/src/main/java/com/swirlds/common/formatting/StringFormattingUtils.java

```diff
@@ -201,4 +202,27 @@ public static String commaSeparatedNumber(final double value, final int decimalP
         commaSeparatedNumber(sb, value, decimalPlaces);
         return sb.toString();
     }
+
+    /**
+     * Sanitize a timestamp to a string that is save to use in a file name. Replaces ":" with "+".
+     *
+     * @param timestamp
+     * 		the timestamp to sanitize
+     * @return the sanitized timestamp
+     */
+    public static String sanitizeTimestamp(final Instant timestamp) {
```

Comment:
@NonNull for timestamp  param and return value + null check for timestamp

Reply:
I can't until this merges: https://github.com/hashgraph/hedera-services/pull/5481